Training_node/first_project/test/users.js

```diff
@@ -368,3 +376,48 @@ describe('/users/:user_id/albums GET', () => {
     });
   });
 });
+
+describe('/users/albums/:id/photos GET', () => {
+  beforeEach(() => {
+    nock(config.common.urlRequests.base)
+      .get('/photos?albumId=2')
+      .reply(200, onePhoto);
+  });
+  it('should be successful', done => {
+    successfulLogin().then(res => {
```

Comment:
do we have a test for admins?

Reply:
Yes! The function sucessfulLogin() if it dont receive params (email), log in with admin.